package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "homepage": "https://apps.rsgis.dev/water",
+  "proxy": "https://api.rsgis.dev/development",
```

Comment:
What run config will make the app use the proxy value? Or was this left in by mistake? Just curious, because if I run the app with `npm run mock`, it still uses the mock JSON files (which is what we want), so just wondering where this proxy value comes into play.

Reply:
I put it in mainly for me. If you go into bundle-utils and on line 26 remove the `http://localhost:3030` to just ``, then when you run `npm start` it'll run the frontend against the dev api. I'm not able to run the backend locally so this is helpful to make sure the code will work properly against actual data (I always revert the bundle-utils before pushing up the code). So far I've been manually adding and removing the proxy every time but I figured it doesn't hurt if we leave it in.

Reply:
It's only used in dev mode right? Just want to be sure if we leave it in, it won't do anything weird when the app is running in production.

Reply:
Yea it's only in dev mode